src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/SpgInvoiceRunChargeBreakdownTests.cs

```diff
@@ -317,6 +319,34 @@ public async Task Execute_ReturnsProducer_CountryForCompany()
                 Assert.Single(result);
                 Assert.Equal("UK - England", result.Single().RegOfficeOrPBoBCountry);
             }
-        } 
+        }
+
+        [Fact]
+        public async Task Execute_ReturnsProducer_HasAnnualCharge_ReturnsNo()
```

Comment:
Sorry missed this one, should there be a yes test? :)

Reply:
I'll set one up